processor/k8sattributesprocessor/README.md

```diff
@@ -193,7 +193,8 @@ the processor associates the received trace to the pod, based on the connection
     "k8s.pod.name": "telemetrygen-pod",
     "k8s.pod.uid": "038e2267-b473-489b-b48c-46bafdb852eb",
     "container.image.name": "telemetrygen",
-    "container.image.tag": "latest"
+    "container.image.tag": "0.112.0", 
+    "container.image.repo_digests": "ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen@sha256:b248ef911f93ae27cbbc85056d1ffacc87fd941bbdc2ffd951b6df8df72b8096"
```

Comment:
@ChrsMark is [container.image.repo_digests](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/container.md#container-image-repo-digests) supposed to be a list of strings?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/k8sattributesprocessor/metadata.yaml#L106

that's interesting. it does say it's a slice here. but all of the tests (including before this MR was made) generate a single string. maybe a documentation error?

Reply:
It is a list of strings yes. It should be fixed in that example.

It is also a list of strings in the test at https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/7ff7d1e6fbe0cca7a59f9fe3b69e38b28a7e7361/processor/k8sattributesprocessor/processor_test.go#L1209 (as it should be).

@spiffyy99 could you share in which tests it is a single string?

Reply:
I think this check can just catch the slice case as well. See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/5ce01a357fcc0f1ed47112c9e2b5165f02c751f9/processor/k8sattributesprocessor/e2e_test.go#L1200-L1204.

https://go.dev/play/p/Lq2hbyPl4cj

So I would say that since the example of this PR is updated now we should be fine.

Reply:
thanks for clarifying.